web/libs/editor/src/components/ImageView/ImageView.jsx

```diff
@@ -689,7 +689,6 @@ export default observer(
       }
 
       item.freezeHistory();
-      item.setSkipInteractions(false);
```

Comment:
This prevented `onClick` from getting the right `skipInteractions` flag state. 
It will be set on `mousedown` anyhow so we do not need to reset it here.